app/domain/authentication/authn_jwt/validate_input.rb

```diff
@@ -0,0 +1,56 @@
+require 'command_class'
+
+# This class
+module Authentication
+  module AuthnJwt
+
+    ValidateInput ||= CommandClass.new(
+      dependencies: {
+        # ValidateWebserviceIsWhitelisted is also calling ValidateAccountExists
+        # it's better to be explicit and on the safe side
+        validate_account_exists: ::Authentication::Security::ValidateAccountExists.new,
+        validate_webservice_is_whitelisted: ::Authentication::Security::ValidateWebserviceIsWhitelisted.new,
+        jwt_validate_body: Authentication::AuthnJwt::ValidateRequestBody.new
+      },
+      inputs: %i[authentication_parameters enabled_authenticators]
```

Comment:
Just an idea: WDYT that `authentication_parameters` will contains `enabled_authenticators` because this is an interface function so it should be with the same signature like others.

Example to the code:
```
      inputs: %i[authentication_parameters]
    ) do
      extend(Forwardable)
      def_delegators(:@ authentication_parameters, :enabled_authenticators)
```

Reply:
i would leave the `authentication_parameters` to include only data of the current authn request.

Reply:
Agree this parameter has been removed

Reply:
what have been removed?